tezos/context/src/lib.rs

```diff
@@ -113,6 +113,9 @@
 //! functionality that interacts with the repository (commit and checkout).
 //!
 
+#![feature(cow_is_borrowed)]
```

Comment:
Ouch, is this needed? cannot it be implemented in our own code instead?

Reply:
Looking at the diff I don't see where it is used btw.

Reply:
Thanks, it's not used I forgot to remove that line